src/plugins/dashboard/public/dashboard_app/top_nav/add_panel_action_menu_items.ts

```diff
@@ -26,25 +30,53 @@ const onAddPanelActionClick =
     } else action.execute(context);
   };
 
+export type GroupedAppPanelActions = EuiContextMenuPanelDescriptor & {
```

Comment:
@darnautov Could it be a typo and you meant to use `GroupedAddPanelActions` instead?

Reply:
fixed in https://github.com/elastic/kibana/pull/179872/commits/dce99fa0afe39bbe5390ffef0c9d66255e6624ff

Reply:
Great work here! I'll soon start with our embeddables. Looks like you will have covered all our needs already :-)